.github/workflows/e2e-repo-main.yaml

```diff
@@ -0,0 +1,56 @@
+name: e2e - run cypress tests for PRs/pushes
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "cypress/**"
+
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "cypress/**"
+      - ".github/workflows/e2e*.yaml"
+
+concurrency:
+  group: e2e-repo-main-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  extract-test-tags:
+    runs-on: ubuntu-latest
+    outputs:
+      test_tags: ${{ steps.extract.outputs.test_tags }}
+    steps:
+      - name: Extract test tags from PR body
+        id: extract
+        env:
+          PR_BODY: ${{ github.event.pull_request.body }}
+          DEFAULT_TAGS: "@ci,@tier0"
+        run: |
+          if [[ -n "$PR_BODY" ]]; then
+            # Look for pattern "ci test tags: @tag1,@tag2,..." in PR body
+            EXTRACTED=$(echo "$PR_BODY" | grep -oP '^ci test tags: \K@.+(,@.+)*$' || true)
```

Comment:
Suggestion : 

This regex Match "ci test tags:" anywhere in the body (case-insensitive) and  Remove spaces → allow "@ci, @tier0"

```
EXTRACTED=$(echo "$PR_BODY" | grep -ioP 'ci test tags:\s*\K@.*' || true)

EXTRACTED=$(echo "$EXTRACTED" | tr -d ' ')
```